src/App.vue

```diff
@@ -1,33 +1,54 @@
 <template>
     <div id="app">
-        <Header />
-        <Readout />
+        <Header v-if = 'header_status'/>     
+        <div class = 'main_content'> 
+            <Main @hideHeader = '_hideHeader(false)'> </Main>
+        </div> 
         <Footer />
     </div>
 </template>
 
 <script>
-import Readout from "@/components/Readout.vue";
+import Main from "@/components/Main.vue";
 import Header from "@/components/Header.vue";
 import Footer from "@/components/Footer.vue";
 
+
 export default {
     name: "App",
     components: {
         Header,
         Footer,
-        Readout
+        Main
+    },
+    data() {
+        return {
+            header_status: true
```

Comment:
Aren't we using camelCase in this project? So this should not be `header_status` but rather `headerStatus`.